### DISCUSSION TITLE
Comerciais Gringos

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#473] [Vídeos]
Post de referência: https://www.vish.blog.br/videos/comerciais-engracados/

**Comment by angellisy** [comment]
Omodeoso! Os salsichinhas fantasiados de salsicha! :heart\_eyes:

**Comment by one-eyed-willy** [comment]
[![Wazzup!!!](https://i.vimeocdn.com/filter/overlay?src0=https%3A%2F%2Fi.vimeocdn.com%2Fvideo%2F1442275823-853e38ab96853310012b433d39ad0ac693f6b6e5c3fe6122f7519d6246a21347-d_1280x528&src1=https%3A%2F%2Ff.vimeocdn.com%2Fimages_v6%2Fshare%2Fplay_icon_overlay.png)](https://player.vimeo.com/video/715642452 "Wazzup!!!")
